### PR TITLE
[NETBEANS-2443] Fix-Can't start profiler windows

### PR DESCRIPTION
**Pull request by moacirrf**
https://issues.apache.org/jira/browse/NETBEANS-2443

Fix start of profiler on windows the bug was caused by wrong command  generated, so i compare both commands:
image

After the change 
image

I try don't mess thing so much, because i don't understand why so much lines are generated, but a tested on Linux and windows.
Run, Debug e Profiler.

### CODE REVIEW

File: java/maven/src/org/netbeans/modules/maven/execute/MavenCommandLineExecutor.java [hunk 1]
```diff
@@ -424,9 +421,8 @@ public void run() {
             }
             //skip envs, these get filled in later.
             //#228901 since u21 we need to use cmd /c to execute on windows, quotes get escaped and when there is space in value, the value gets wrapped in quotes.
-            String value = (Utilities.isWindows() ? entry.getValue().replace(quote, escaped) : entry.getValue().replace(quote, "'"));
+            String value = (Utilities.isWindows() ? entry.getValue().replace(quote, "") : entry.getValue().replace(quote, "'"));
             if (Utilities.isWindows() && value.endsWith("\"")) {
-                //#201132 property cannot end with 2 double quotes, add a space to the end after our quote to prevent the state
```
Comment (ebarboni) [inline java/maven/src/org/netbeans/modules/maven/execute/MavenCommandLineExecutor.java:429]: may no need to remove this comment

**Comment by sdedic** [comment]
The reasons are more deeper: I fear this patch will fix the profiler, but will break other code that may use e.g. system properties that contains spaces or other 'illegal' chars. As noted in [NETBEANS-5732](https://issues.apache.org/jira/browse/NETBEANS-5732), the culprit is the profiler integration which quotes the output of StartupExtender, when it should not.

File: java/maven/src/org/netbeans/modules/maven/execute/MavenCommandLineExecutor.java (hunk 1 above)
Comment (moacirrf) [reply java/maven/src/org/netbeans/modules/maven/execute/MavenCommandLineExecutor.java:429]: humm..I will put again this comment

**Comment by moacirrf** [comment]
> The reasons are more deeper: I fear this patch will fix the profiler, but will break other code that may use e.g. system properties that contains spaces or other 'illegal' chars. As noted in [NETBEANS-5732](https://issues.apache.org/jira/browse/NETBEANS-5732), the culprit is the profiler integration which quotes the output of StartupExtender, when it should not.

Yes, is possible...i will test with netbeans on path with spaces.
Thanks

**Pull request closed without merging**